packages/tools/devtools/devtools/README.md

```diff
@@ -29,10 +29,75 @@ The Devtools' API surface is designed to fit nicely into most application flows.
+logger is the way in which that telemetry gets into Fluid Devtools.
+
+You can obtain a `DevtoolsLogger` by calling its constructor, and then should pass it to `initializeDevtools` _and_
+to your application.
+This way the logger will receive all the telemetry generated by Fluid Framework in your application, and forward it to
+Fluid Devtools as necessary.
+For example, when using the `AzureClient` API you would do something like this (note how `devtoolsLogger` is passed
+to `initializeDevtools()` _and_ to `new AzureClient()`):
+
+```typescript
+import { DevtoolsLogger, initializeDevtools } from "@fluid-experimental/devtools";
+
+// Instantiate the logger
+const devtoolsLogger = new DevtoolsLogger();
+
+// Pass the logger when instantiating the AzureClient
+const clientProps = {
+    connection: { ... }, // Your application's configuration to connect to the Fluid service
+    logger: devtoolsLogger,
+};
+const client = new AzureClient(clientProps);
+
+// Use the AzureClient to create a Container
+const containerSchema = { /* Define the objects in your Fluid Container */ };
+const { container, services } = await client.createContainer(containerSchema)
+
+// Initialize the Devtools passing the logger and your Container.
+// The Container could be added later as well with devtools.registerContainerDevtools().
+const devtools = initializeDevtools({
+    logger: devtoolsLogger,
+    initialContainers: [
+        {
+            container,
+            containerKey: "My Container",
+        },
+    ],
+});
+```
+
+If you're not working with `AzureClient` but with lower-level APIs (e.g. you manually instantiate a `Loader` from the
+[@fluidframework/container-loader package][container-loader-package] package, you probably want to refer to
+the [@fluid-experimental/devtools-core package][devtools-core-package] instead of this one).
+
+During local development the recommendation is that your application should receive the `DevtoolsLogger` instance _instead_
+of any logger it would normally receive when deployed to a real environment, to avoid local development activity from
+mixing with real telemetry.
+If you still want to provide a real application logger and use the Fluid Devtools features that are powered by telemetry
+at the same time, you can pass an existing logger to the `DevtoolsLogger` constructor and it will forward all telemetry
+it receives to that logger as well:
+
+```typescript
+import { DevtoolsLogger } from "@fluid-experimental/devtools";
 
-See [here](https://fluidframework.com/docs/apis/devtools#initializedevtools-function) for associated API documentation.
+// Your application's logger
+const yourApplicationLogger = getInstanceOfYourApplicationLogger();
+
+const devtoolsLogger = new DevtoolsLogger(yourApplicationLogger);
+
+// Pass devtoolsLogger to initializeDevtools() and to your application as described above
+```
 
 ### Clean-up
 
```

Comment:
All your changes are good. But I do have a suggestion for the "Working in the package" title below. I find "Working in the package" to be confusing - I think it's referring to contributing to changes in the package right? Maybe rename it to "Contributing to the package"? Could do in a different PR too since that change is unrelated to your additions above!

Reply:
Yeah, that's what that section is about. We can make that change in a separate PR as you suggest.

Reply:
Had to make another fix so I went ahead and updated the header too 👍